Marlin/ultralcd.h

```diff
@@ -15,6 +16,10 @@
   void lcd_reset_alert_level();
   bool lcd_detected(void);
 
+  #ifdef LCD_USE_I2C_BUZZER
+    void lcd_buzz(long duration, uint16_t freq);
```

Comment:
Maybe "i2c_buzz" is better, since then we can think about buzzers without thinking about LCDs.